turbo.json

```diff
@@ -8,22 +8,30 @@
         "!{pkgs/query,discord-bot,query-api,ui}/**/{build,dist,node_modules,.next}/**"
       ]
     },
-    "//#_format": {
+    "//#format.prettier": {
       "inputs": [
         "*.{cjs,js,jsx,ts,tsx,yaml,yml,json,md,mdx,css,html,graphql}",
-        "{.github,pkgs,circom,discord-bot,query-api,ui}/**/*.{cjs,js,jsx,ts,tsx,yaml,yml,json,md,mdx,css,html,graphql}",
-        "!{pkgs,circom,discord-bot,query-api,ui}/**/{build,dist,node_modules,.next}/**"
+        "{.github,pkgs,circom,discord-bot,query-api,test,ui}/**/*.{cjs,js,jsx,ts,tsx,yaml,yml,json,md,mdx,css,html,graphql}",
+        "!{pkgs,circom,discord-bot,query-api,test,ui}/**/{build,dist,node_modules,.next}/**"
       ]
     },
-    "_format": {
+    "format.cargo": {
+      "cache": false,
+      "inputs": ["pkgs/{merkle-tree,spartan-ecdsa}-wasm/src/**/*.rs"]
+    },
+    "format.cargo.fix": {
+      "cache": false,
+      "inputs": ["pkgs/{merkle-tree,spartan-ecdsa}-wasm/src/**/*.rs"]
+    },
+    "format.forge": {
       "cache": false,
```

Comment:
Somehow `turbo` isn't detecting file changes for `.sol` files.
So need to set cache to `false` to avoid wrongly hitting it.